File: beginners/2-problemset-condicionais/1046-tempo-jogo/tempo_jogo.cpp
#include <iostream>
using namespace std;

int main() {
    int h_ini, h_fim, total;
    int min = 1, max = 24;

    cin >> h_ini >> h_fim;

    if (h_ini > h_fim)
        total = (max - h_ini) + h_fim;
    else if (h_ini < h_fim)
        total = (min + h_fim) - h_ini - min;
    else
        total = max;

    cout << "O JOGO DUROU" << " " << total << " " << "HORA(S)" << endl;

    return 0;
}

File: beginners/2-problemset-condicionais/Python/1046-tempo-jogo/tempo_jogo.py
h_ini, h_fim = input().split()
h_ini = int(h_ini)
h_fim = int(h_fim)

if h_ini > h_fim:
    tempo_total = 24 - h_ini + h_fim
elif h_ini < h_fim:
    tempo_total = h_fim - h_ini
else:
    tempo_total = 24

print('O JOGO DUROU', tempo_total, 'HORA(S)')
